web/src/components/shared/AirgapRegistrySettings.tsx

```diff
@@ -121,6 +132,19 @@ class AirgapRegistrySettings extends Component<Props, State> {
             rewriteMessage: registryDetails.error,
           });
         } else {
+          this.setState({
+            originalRegistry: {
+              hostname: hostname,
+              username: username,
+              password: password,
+              namespace: namespace,
+            },
+            hostname: hostname,
+            username: username,
+            password: password,
+            namespace: namespace,
+            isReadOnly: isReadOnly,
```

Comment:
```suggestion
              hostname,
              username,
              password,
              namespace,
            },
            hostname,
            username,
            password,
            namespace,
            isReadOnly,
```
Minor suggestion for future stuff. This is a shortcut for assigning variables to an object. It will use the variable name as the property name.